platzky/feature_flags_wrapper.py

```
@@ -0,0 +1,84 @@
+        raw_data: dict[str, bool],
+    ) -> None:
+        super().__init__(raw_data)
+        # Use object.__setattr__ to bypass __getattr__ override
+        object.__setattr__(self, "_enabled_flags", enabled_flags)
+
+    def __getitem__(self, key: str) -> bool:
+        """Dict bracket access with deprecation warning."""
+        _warn_dict_access()
+        return super().__getitem__(key)
+
+    def get(self, key: str, default: bool | None = None) -> bool | None:  # type: ignore[override]
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)